### DISCUSSION TITLE
See $ amount before sending a transaction

### DISCUSSION THREAD

**Discussion by Waldi28** [alephium/alephium-frontend#432] [Feature requests]
Sent a transaction few minutes ago to pay someone 40 dollar in $ALPH, just a suggestion but it would be nice in the future to see the dollar amount when i type the amount of alph before i send it.

Otherwise, i need to look at coinmarketcap or something else, and then have to use a calculator.

**Comment by nit27** [comment]
Good idea!
Maybe nice to have a switch between fiat & ALPH (or other tokens) while enter the amount about to send. For example, enter 1 ALPH shows xxx $US or enter xxx $US shows 1 ALPH). With this flow, we should have a `refesh button` for updating the price and/or amount of tokens.

**Comment by nop33** [comment]
This feature is implemented in this Pull Request:
- https://github.com/alephium/alephium-frontend/pull/1248
Closing discussion
